app/controllers/home_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
     load_graphs_for_admin
     load_waiting_applications
     @search_form = Forms::Search.new
+    @state = DwpMonitor.new.state
```

Comment:
Can you add this assignment to the controller spec as there's no other place we test that it's there?